conjur/wrapper/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+wrapper package
+
+This package contain classes that are wrapping third party libraries
```

Comment:
This module contains wrapper classes for third party libraries

Reply:
Changed